src/dos/dos_files.cpp

```diff
@@ -935,8 +935,8 @@ static bool isvalid(const char in){
 #define PARSE_RET_BADDRIVE      0xff
 
 // TODO: Refactor and document this function until it's understandable
-uint8_t FCB_Parsename(uint16_t seg,uint16_t offset,uint8_t parser ,char *string, uint8_t *change) {
-	char * string_begin=string;
+uint8_t FCB_Parsename(uint16_t seg,uint16_t offset,uint8_t parser ,const char *string, uint8_t *change) {
+	const char * string_begin=string;
 	uint8_t ret=0;
```

Comment:
If you're able spit into more commits, running `./scripts/format-commit.sh` on each commit will get these lines formatted a bit better too, given we're touching them (it will add spaces between `=`)